bin/ruby-build

```diff
@@ -9,7 +9,7 @@
 #   -v/--verbose     Verbose mode: print compilation status to stdout
 #   -4/--ipv4        Resolve names to IPv4 addresses only
 #   -6/--ipv6        Resolve names to IPv6 addresses only
-#   --definitions    List all built-in definitions
+#   --definitions    List all available definitions
```

Comment:
I'm concerned that "available" implies ruby-build will be checking online sources, or is otherwise aware of all ruby releases. That has actually been a point of confusion in the past.

Perhaps "all locally available definitions" ?

Reply:
Good point! I'd be also fine with `all local definitions`

Reply:
Thanks for the feedback. Changed to "local".